pkgs/development/python-modules/cffi/default.nix

```
@@ -57,6 +48,11 @@ else
         --replace '/usr/include/libffi' '${lib.getDev libffi}/include'
     '';
 
+    disabledTests = lib.optionals stdenv.isFreeBSD [
```

```suggestion
    disabledTests = lib.optionals stdenv.hostPlatform.isFreeBSD [
```

see https://github.com/NixOS/nixpkgs/pull/341407